daprdocs/content/en/operations/configuration/increase-read-buffer-size.md

```diff
@@ -0,0 +1,60 @@
+---
+type: docs
+title: "How-To: Handle large http read buffer size"
+linkTitle: "Http read buffer size"
```

Comment:
HTTP header size

Reply:
Thank you for your reply, I will make the appropriate changes.